packages/components/src/components/SquiggleEditor.tsx

```diff
@@ -54,17 +55,13 @@ export const SquiggleEditor: React.FC<SquiggleEditorProps> = (props) => {
     width,
     height = 200,
     enableLocalSettings = false,
+    continues = defaultContinues,
```

Comment:
`undefined` would be easier, `useSquiggle` would take it and upgrade to the array. (I changed this yesterday)